elyra/pipeline/component_catalog.py

```diff
@@ -25,25 +24,24 @@
 from jinja2 import Environment
 from jinja2 import PackageLoader
 from jinja2 import Template
-from traitlets import default
-from traitlets import Integer
-from traitlets.config import LoggingConfigurable  # noqa: H306 (alphabetical order catch-22)
+from traitlets.config import SingletonConfigurable
 
 from elyra.metadata.manager import MetadataManager
 from elyra.metadata.metadata import Metadata
 from elyra.metadata.schemaspaces import ComponentCatalogs
+from elyra.pipeline.airflow.component_parser_airflow import AirflowComponentParser  # noqa F401 - TODO
```

Comment:
Since you're going thru `__subclasses__`, I'm surprised you need these import statements.  Do things break at runtime otherwise?  (I suppose they do since the "get_instance" method needs to create their instance.)

Since the `parse_class_map` is essentially static and (will be) initialized once, it might be best to just reference the subclasses directly when building the map and add that map's update to the list of things folks need to do when _contributing_ a new runtime type (along with a complete component parser :smile: ).